btk/sampling_functions.py

```diff
@@ -5,6 +5,8 @@
 import astropy
 import numpy as np
 from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from fast3tree import find_friends_of_friends
```

Comment:
need to add this new package to the `pyproject.toml` and `poetry.lock`, if you don't know how to do this let me know and I can add it

Reply:
Yeah, it would be great if you could help me with that. Not sure how to do it